### ISSUE TITLE
Don't touch NULL values?

### ISSUE THREAD

**Issue by spyro2000** [kibitan/masking#59]
Hi there,

loving your tool :)
Is is possible to **only replace values which are actually present**? If some value is in fact _null_ it should just stay this way so that the original structure of the data is kept?

Something like

```yml
my_table:
   user_comment: ?? Placeholder comment # <-- read as 'if not null, insert "Placeholder comment"'
````

Thank you in advance,
spyro

**Comment by kibitan** [comment]
Hi @spyro2000 thank you so much for your comment, sorry for the late reply. 
It is actually a good idea :+1: I had a similar feature in my mind. 

I will work on it, it may take time to catch my time. I also appreciate it if you send a pull request, free to do so.
I'll keep you posted, Thank you!

best regards,
Chikahiro

**Comment by kibitan** [comment]
@spyro2000 it is implemented on v1.1.1 👍

**Comment by spyro2000** [comment]
@kibitan NICE!
